README.md

```diff
@@ -45,34 +47,39 @@ target_os = []
-require network access to Google's code/storage infrastructure. Set `ssl_root` to the directory of your OpenSSL 1.1.0 binary. Set `msdk_root` to the 
-directory of your Intel Media SDK for Windows, version 2018 R1 or higher. Use `--gn_gen` to generate args.gn during the first build or when
-you change the `msdk_root` or `ssl_root` paths.
-- You can also specify `quic_root`  to the path containing QUIC library built from owt-deps-quic. This will build the SDK with QUIC enabled for conference mode.
+1. Set **DEPOT_TOOLS_WIN_TOOLCHAIN** to **0** in your system environment.
+1. Set Visual Studio related environmental variables. You can either call `VsDevCmd.bat` under `Common7\Tools` directory of your Visual Studio installation, or start build in `Developer Command Prompt for VS`.
+1. In `src` directory, Run `gclient sync`. It may take long time to download large amount of data the first time you run it. Also you are required to run this whenever `DEPS` file is updated.
+1. Go to the `src/scripts` directory, and run: `python build-win.py --gn_gen --sdk --tests --ssl_root /path/to/ssl --msdk_root /path/to/msdk --output_path /path/to/output`.
+  - The optional `msdk_root` should be set to the directory of your Intel MediaSDK for Windows, version 2020 R1 or higher. This is typically
+  `C:\Program Files (x86)\IntelSWTools\Intel(R) Media SDK 2020 R1\Software Development Kit`. If specified, will enable hardware accelerated video codecs for most of the video codecs.
+  - The optional `--sdk` is to inform the build script to use `lib.exe` that is part of Visual Studio toolchain for merging owt libraries with external openssl libraries.
+
 
 #### Linux
-- Run `gclient sync`. It may take a long time to download a large amount of data.
-- Go to the `src/scripts` directory, and run: `python build_linux.py --gn_gen --sdk --tests --ssl_root /path/to/ssl --output_path /path/to/out`. 
-  The built binary will be under `output_path`, the document for sdk will also be copied to this directory if docs have been generated. 
-If `output_path` is not set, the built binary will be under the `src/out` directory. Note the first time you run this it will take a 
-long time to pull chromium/webrtc dependencies and require network access to Google's code/storage infrastructure. Set `ssl_root` to the 
-directory of your OpenSSL 1.1.0 binary. Use `--gn_gen` to generate args.gn during the first build or when you change the `ssl_root` path.
-If `--msdk_root` is specified to correct Intel MediaSDK path, for example, '/opt/intel/mediasdk', hardware codecs will be built besides the
-software implementations. If `--fake_audio` is specified, the internal audio devices implementation based on alsa or pulseaudio will not be built.
+1. In `src` direcotry, run `gclient sync`. It may take long time to download large amount of data.
+1. Go to the `src/scripts` directory, and run: `python build_linux.py --gn_gen --sdk --tests --ssl_root /path/to/ssl --output_path /path/to/out`.
```

Comment:
One more step is needed on Linux.

Run https://chromium.googlesource.com/chromium/src/+/HEAD/build/install-build-deps.sh to install dependencies.

Reply:
This is already covered in Prepare the development environment-> WebRTC stack build dependencies link in the same document.